content/applications/general/digest_emails/customize_periodic_digest.rst

```diff
@@ -0,0 +1,20 @@
+=====================================
+Customize *Your Odoo Periodic Digest*
+=====================================
+
+To cuzomize the standard *Your Odoo Periodic Digest* email, find the **Digest Email** heading under the **Statistics** section in :menuselection:`Settings --> General Settings`. Then click the **[-->]** icon next to the dropdown selection. 
```

Comment:
Here's a suggestion to illustrate how you could write this more concisely:

```suggestion
To edit a **Digest Email**, go to :menuselection:`Settings --> General Settings --> Statistics -->
Digest Email`, and click on the *external link* next to the dropdown selection list.
```

However, you probably won't use this as what this does is merely open the selected Digest Email currently in use.

If you squash your "customize" and "create" sections, you'll probably explain only once how to open the **Digest Emails** menu from where admins can either open or edit digest emails.